src/theme/components/DisplayCard.ts

```diff
@@ -0,0 +1,104 @@
+import { ComponentMultiStyleConfig } from "@chakra-ui/theme"
+import { anatomy, PartsStyleObject } from "@chakra-ui/theme-tools"
+
+export const DISPLAY_CARD_THEME_KEY = "displayCard"
+
+// Determines whether the display card has content, or only contains the title,
+// subtitle, icon and any action button, or only contains the content
+export type DisplayCardVariant =
+  | "onlyHeader"
+  | "onlyContent"
+  | "headerAndContent"
+
+export const parts = anatomy(DISPLAY_CARD_THEME_KEY).parts(
+  "container",
+  "header",
+  "title",
+  "caption",
+  "content",
+  "footer"
+)
+
+const variantHasContent: PartsStyleObject<typeof parts> = {
+  header: {
+    paddingBottom: "0.5rem",
+  },
+  content: {
+    display: "flex",
+    overflow: "auto",
+    justifyContent: "flex-start",
+    alignItems: "flex-start",
+    textAlign: "left",
+  },
+}
+
+const variantNoContent: PartsStyleObject<typeof parts> = {}
+
+const variantOnlyContent: PartsStyleObject<typeof parts> = {
+  container: {
+    paddingY: "1rem",
+  },
+  content: {
+    display: "flex",
+    overflow: "auto",
+  },
+}
+
+export const DisplayCard: ComponentMultiStyleConfig = {
+  parts: parts.keys,
+  baseStyle: {
+    container: {
+      display: "flex",
+      justifyContent: "flex-start",
+      boxSizing: "border-box",
+      paddingInlineEnd: "2rem",
+      paddingInlineStart: "2rem",
+      paddingTop: "2rem",
+      paddingBottom: "1.5rem",
+      border: "1px solid",
+      borderColor: "border.action.light",
+      borderRadius: "4px",
+      w: "100%",
+      overflow: "auto",
+      alignItems: "start",
+    },
+    header: {
+      display: "flex",
+      alignItems: "start",
+      w: "100%",
+    },
+    title: {
+      textStyle: "h3",
+      marginRight: "0.625rem",
+      paddingBottom: "0",
+    },
+    caption: {
+      textStyle: "caption-2",
+      paddingTop: "0.25rem",
+      color: "text.helper",
+    },
+    content: {
+      overflow: "auto",
+      justifyContent: "flex-start",
+      display: "flex",
+      alignItems: "flex-start",
+      textAlign: "left",
+      marginTop: "0rem",
+      w: "100%",
+    },
+    footer: {
+      display: "flex",
+      paddingTop: "1.625rem",
+      paddingBottom: "0.5625rem",
+      w: "100%",
+    },
+  },
+  variants: {
+    hasContent: variantHasContent,
+    noContent: variantNoContent,
+    onlyContent: variantOnlyContent,
```

Comment:
As discussed, lets change the naming of these variants, `onlyContent` and `hasContent` seem confusing!

Reply:
Adjusted in 61c67d7!